### DISCUSSION TITLE
Any updates on fixing the jackson-databind CVE?   thx

### DISCUSSION THREAD

**Discussion by SteveS7777** [eclipse-rdf4j/rdf4j#4125] [General]
Any updates on fixing the jackson-databind CVE?

**Comment by hmottestad** [comment]
I guess it's just a matter of updating the relevant dependencies. If we need to upgrade to a newer bug fix version then that should be fine, if it's a minor or major version we have to do IP due diligence. 

Would you be interested in creating an issue and contributing a PR?

**Reply by SteveS7777** [reply]
we did do that and rebuilt it from source on our side.  Not sure if we can do the PR. I wil check.  thank you
